### PR TITLE
Add wildcard to .tfvars in .gitignore

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

Add a wildcard to the `.tfvars` entry in the `.gitignore` file.

## 💭 Motivation and context ##

This ensures that `.tfvars` files in subdirectories are also ignored by git, as we intended.

## 🧪 Testing ##

Verified that `tfvars` files in subdirectories no longer show up in `git status` output.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,4 @@ dist
 ## Terraform ##
 .terraform
 .terraform.lock.hcl
-.tfvars
+*.tfvars
```
Comment (mcdonnnj) [inline .gitignore:22]: A bit of scope creep but this would fully align us with the [`.gitignore` file in cisagov/skeleton-tf-module](https://github.com/cisagov/skeleton-tf-module/blob/develop/.gitignore).

```suggestion
terraform.tfstate
terraform.tfstate.backup
*.tfvars
```

**Review by mcdonnnj — APPROVED** [review]
Glad you're fixing this oversight. I had one suggestion for your consideration.
